views/publication/summary/summary.templ

```diff
@@ -132,27 +132,27 @@ templ Summary(c *ctx.Ctx, args SummaryArgs) {
 							</ul>
 							// TODO: pass on args.Target
 							@contributorviews.Summary(c, "author", args.Publication.Author, views.URL(args.URL).SetQueryParam("show", "contributors").String())
+							<div class="d-flex align-items-center">
+								<i class="if if-building if--small if--muted me-2"></i>
+								if len(args.Publication.RelatedOrganizations) > 0 {
+									// TODO: pass on args.Target
+									@relatedorganizationviews.Summary(c, args.Publication.RelatedOrganizations, views.URL(args.URL).SetQueryParam("show", "contributors").String())
+								} else {
+									<a
+										href={ views.URL(args.URL).SetQueryParam("show", "contributors").SafeURL() }
+										class="c-link-muted"
+										if args.Target != "" {
+											target={ args.Target }
+										}
+									>
+										<i class="if if-edit if--small"></i>
+										<em>Add department</em>
+									</a>
+								}
+							</div>
 						</div>
-						<div class="collapsible-card-content collapse" data-collapsible-card-content>
+						if vabb := args.Publication.VABB(); vabb != "" || args.Publication.Legacy || len(args.Publication.RelatedDataset) > 0 {
```

Comment:
Not sure if this is the best solution to hide the block when nothing is in there.

Reply:
this won't work correctly
the conditions ` || args.Publication.Legacy || len(args.Publication.RelatedDataset) > 0` will only be evaluated when `vabb != ""`

Reply:
better add a helper function like `func showX(p *models.Publication) bool`